apps/framework-cli/src/cli/routines/format_query.rs

```diff
@@ -56,8 +57,12 @@ fn parse_sql(sql: &str) -> Result<Vec<Statement>, RoutineFailure> {
 ///
 /// * `Result<(), RoutineFailure>` - Ok if valid, error with helpful message if invalid
 pub fn validate_sql(sql: &str) -> Result<(), RoutineFailure> {
-    parse_sql(sql)?;
-    Ok(())
+    validate_sql_statement(sql).map_err(|e| {
```

Comment:
I believe `sqlparser` rejects some CH SQL statements. Maybe use its error message when CH rejects it?